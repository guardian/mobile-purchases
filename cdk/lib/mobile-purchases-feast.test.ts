import { App } from "aws-cdk-lib";
import { Template } from "aws-cdk-lib/assertions";
import { MobilePurchasesFeast } from "./mobile-purchases-feast";

describe("The MobilePurchasesFeast stack", () => {
  it("matches the snapshot", () => {
    const app = new App();
    const stack = new MobilePurchasesFeast(app, "MobilePurchasesFeast", { stack: "mobile", stage: "TEST" });
    const template = Template.fromStack(stack);
    expect(template.toJSON()).toMatchSnapshot();
  });
});
