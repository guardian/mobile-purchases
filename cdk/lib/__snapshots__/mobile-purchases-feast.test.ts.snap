// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The MobilePurchasesFeast stack matches the snapshot 1`] = `
{
  "Metadata": {
    "gu:cdk:constructs": [],
    "gu:cdk:version": "TEST",
  },
}
`;
